llumnix/backends/vllm/scheduler.py

```diff
@@ -223,29 +211,17 @@ def _get_instance_info(self) -> InstanceInfo:
             num_blocks_first_waiting_request=num_blocks_first_waiting_request,
             waiting_time_first_waiting_request=waiting_time_first_waiting_request,
             num_blocks_all_waiting_requests=num_blocks_all_waiting_requests,
-            inference_type=BackendInferenceType.PREFILL if self.prefilling_seq_groups \
-                           else BackendInferenceType.DECODE,
         )
         for seq_group in self.running:
             instance_info.running_seq_lens.extend([seq.get_len() for seq in seq_group.get_seqs()])
             instance_info.num_seqs = len(instance_info.running_seq_lens)
-        # TODO(s5u13b): Only correct when using prefill preemption batching policy.
-        instance_info.num_batched_tokens = sum([seq_group.get_seqs()[0].get_len() for seq_group in self.prefilling_seq_groups])\
-                                         if self.prefilling_seq_groups else len(instance_info.running_seq_lens)
+        if self.running:
+            instance_info.inference_type = self.running[-1].inference_type
+        instance_info.num_batched_tokens = sum([seq_group.request_len for seq_group in self.running])\
+                                         if instance_info.inference_type == RequestInferenceType.PREFILL else len(instance_info.running_seq_lens)
```

Comment:
Is num_batched_tokens correct when using chunked-prefill? If not, we can add a todo here.

Reply:
It's wrong in chunked-prefill,  added todo.